### PR TITLE
Here's Trouble Plugin

### PR DESCRIPTION
**Pull request by AmIRighteous**
Plugin that plays a sound clip/overhead text of someone saying "Here's Trouble" whenever a friend or clanmate comes into view of the player.

This is also my first plugin attempt so if I've done something incorrectly please let me know!

### CODE REVIEW

**Comment by AmIRighteous** [comment]
Fixed package name to com.herestrouble.HeresTroublePlugin in plugin repo

File: plugins/heres-trouble [hunk 1]
```diff
@@ -0,0 +1,2 @@
+repository=https://github.com/AmIRighteous/heres-trouble.git
+commit=992b3910564a93eeb78e2f58c84c0f7c37b16ac2
```
[the remark this answers is not in the document]
Comment (AmIRighteous) [reply plugins/heres-trouble]: @Felanbird great callout! That would've kickstarted the GHAs as well without me having to rerun them manually, thanks!

**Comment by YvesW** [comment]
Please change your config group to something unique.
I'd also remove the log.info calls to not fill up the user's logs, but that's my own 0.02

**Comment by Felanbird** [comment]
> Plugin that plays a sound clip/overhead text of someone saying "Here's Trouble"

Do you know who this "someone" is? As we can't distribute a sound file of a person we don't have permission from.

**Comment by AmIRighteous** [comment]
> Please change your config group to something unique. I'd also remove the log.info calls to not fill up the user's logs, but that's my own 0.02

Thanks for the callout @YvesW ! Changed the config group as well as removed the `log.info` calls on start/end

**Comment by AmIRighteous** [comment]
> > Plugin that plays a sound clip/overhead text of someone saying "Here's Trouble"
> 
> Do you know who this "someone" is? As we can't distribute a sound file of a person we don't have permission from.

@Felanbird yes it's a friend of mine who gave me permission verbally to use those clips. What is a valid proof of permission for us to use these sound clips?

**Comment by Felanbird** [comment]
> @Felanbird yes it's a friend of mine who gave me permission verbally to use those clips. What is a valid proof of permission for us to use these sound clips?

Sorry for the late reply, a sound clip of your friend saying `you have his permission to use his voice for a Runelite plugin` would likely suffice.

**Comment by AmIRighteous** [comment]
> > @Felanbird yes it's a friend of mine who gave me permission verbally to use those clips. What is a valid proof of permission for us to use these sound clips?
> 
> Sorry for the late reply, a sound clip of your friend saying `you have his permission to use his voice for a Runelite plugin` would likely suffice.

Sounds good! Will reply with that shortly

**Comment by AmIRighteous** [comment]
@Felanbird attached is the recording my friend Ed (the voice of Here's Trouble) sent me for permission. If we need to discuss further with him he doesn't have a github, but he has discord and could reply to questions there
https://github.com/runelite/plugin-hub/assets/32687206/58c69ca5-4006-4fb8-8534-9a5252de8b65

**Pull request merged**